apps/dashboard/src/app/components/charts/bar-chart/bar-chart.component.ts

```diff
@@ -98,9 +98,9 @@ export class BarChartComponent implements AfterViewInit, OnDestroy {
     }
 
     // Tooltip event handlers
-    const mouseover = (data: [string, number], index: number, group: any) => {
+    const mouseover = (event: any, data: [string, number]) => {
```

Comment:
can't we use `MouseEvent` instead of `any`?

Reply:
Fixed